### PR TITLE
solution first draft

### PR DESCRIPTION
**Pull request by scottcreynolds**
@AnnJohn

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,39 @@
...
-  * Last, when a user clicks on one of the orders that the product is in, they should be taken to a order page which shows all the products. That should be backed from a JSON response or orders.
-  * Mention that we have a circular dependency here. Products renders the orders association. Orders wants to render it's products association.
-  * Tell them to solve it by having the Order just show product Ids by using the embed ids. Then they'll need to do a bunch of AJAX requests to get each product Title and price. Give them this documentation: http://www.rubydoc.info/gems/active_model_serializers/0.8.2/ActiveModel/Serializer.embed
-  * Scott ^ that may be a bit of a stretch. What do you think?
+We're going to continue updating our products/orders system to use
+ActiveModel::Serializer for JSON serialization. The solution to the
+previous lab is provided.
+
+Don't forget to run `rake db:seed` for some starter data and lots of
+Latin practice!
```
Comment (pletcher) [inline README.md:15]: Quid est?

File: README.md [hunk 2]
```diff
@@ -1,16 +1,39 @@
...
+
+## Instructions
+
+1. Create an ActiveModel::Serializer for `Product` and update the
+   `products_controller` to use the new serializer in place of the
+existing `to_json` code.
+2. Update the product show page to handle the new JSON.
+3. On the products `index` page, update the `More Info` buttton so that it
```
Comment (pletcher) [inline README.md]: "buttton" -> "button"

File: README.md [hunk 3]
```diff
@@ -1,16 +1,39 @@
...
+   uses the `/products/id.json` route to get both description and
+inventory.
+  * **Note:** The serializer will return a different value for
+    `inventory` than the previous API, so you'll have to handle that.
+4. Update the `ProductSerializer` to include the orders for the product.
+5. Update the `More Info` button on the products `index` page to show a
+   list of orders with `id` and `created_at` in addition to the
+description and inventory.
+6. Update the `OrderSerializer` to include the product names of all
+   products on that order, and display them with the order information
```
Comment (pletcher) [inline README.md:33]: The notes in this section are very helpful

**Comment by pletcher** [comment]
As usual with a lab, I have very little to add. Just that minor typo and a bit of confusion about the "Latin practice!" line.

File: README.md (hunk 1 above)
Comment (scottcreynolds) [inline README.md:15]: lorem ipsum joke

**Pull request merged**